.github/workflows/wpt.yml

```
@@ -261,9 +261,9 @@ jobs:
           base: ${{ github.head_ref }}
           committer: Browser Automation Bot <browser-automation-bot@google.com>
           author: Browser Automation Bot <browser-automation-bot@google.com>
-          commit-message: 'test: update the expectations for PR'
-          title: 'test: update the expectations for PR'
-          body: 'Automatically generated by https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/.github/workflows/wpt.yml'
+          commit-message: 'test: update the expectations for PR ${{ github.event.pull_request.number }}'
+          title: 'test: update the expectations for PR ${{ github.event.pull_request.number }}'
```

```suggestion
          title: 'test:  expectations update the #${{ github.event.pull_request.number }}'
```
I think this would be a simple message as the  `#` in GitHub it is a common pattern in here so lets use it.